src/main/scala/com.socrata.geoexport/http/ExportService.scala

```diff
@@ -77,7 +77,8 @@ class ExportService(sodaClient: UnmanagedCuratedServiceClient) extends SimpleRes
       val reqBuilder = {
         base: RequestBuilder =>
           val sfReq = base
-            .path(Seq("export", s"_${fbf}.soqlpack"))
+            .path(Seq("resource", s"_${fbf}.soqlpack"))
+            .addParameter(("$query", s"select * limit ${Int.MaxValue}"))
```

Comment:
So what will happen if there are more results than integer max value? They just won't be included? Should we be doing paging?

Reply:
2147483647 rows is a lot of rows.  Could use Long.MaxValue instead, though.  Postgresql's perfectly happy with it.

Reply:
If that's what the max value is then, I'm not worried about it as much but then really why we are doing a limit at all?

Reply:
Because there's a default limit (of, IIRC, 1000) if you don't explicitly specify one.

Reply:
(which IMO is a bad default but nonetheless it exists)

Reply:
Should we be paging then? If the goal is reduce bringing down servers, paging here instead of dumping all the things seems like a reasonable strategy. We're still dumping all the things with this change. Just in a different way?